### ISSUE TITLE
Page Options

### ISSUE THREAD

**Issue by NitaAlexandr** [Geczy/WP-Simple-Settings-Framework#12]
Can it be used to create page options/fields?

**Comment by jamesckemp** [comment]
No, you'd be better off using something like Advanced Custom Fields or CMB2

**Comment by NitaAlexandr** [comment]
thanks... but can I extend it for that purpose?

**Comment by jamesckemp** [comment]
I wouldn't have thought so - it's more for global options, rather than meta options.
